springboot/pom.xml

```diff
@@ -16,25 +16,26 @@
 
   <properties>
     <java.version>11</java.version>
-    <spring-cloud.version>2021.0.1</spring-cloud.version>
+    <spring-cloud.version>2021.0.3</spring-cloud.version>
+    <spring-native.version>0.12.1</spring-native.version>
   </properties>
   <dependencies>
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.springframework.cloud</groupId>
-      <artifactId>spring-cloud-function-adapter-aws</artifactId>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-json</artifactId>
```

Comment:
By using `spring-boot-starter-json`, we bring the default Jackson configuration Spring users are used to. No need anymore to configure custom `ObjectMapper`, or annotated classes with Jackson annotations.

The penalty is bringing few more Jackson modules as dependencies.